src/style.css

```diff
@@ -1 +1,70 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Consider adding fallback fonts to improve robustness of your font rendering. In case the primary font fails to load, the browser can use the next available font in the list.